cf-manifest/deployments/000-base-cf-deployment.yml

```diff
@@ -8,6 +8,8 @@ meta:
   releases:
   - name: cf
     version: latest
+  - name: nginx
+    version: latest
```

Comment:
We shouldn't be enabling HTTPS _at all_ in this PR as it's out of scope - it was decided that the final implementation would probably be totally different and will be spiked out.